.github/workflows/validate-pr.yml

```
@@ -258,7 +258,7 @@ jobs:
     - name: Setup GraalVM environment
       uses: olafurpg/setup-scala@v10
       with:
-        java-version: graalvm@19.0.0
+        java-version: graalvm@21.0.0
```

```suggestion
        java-version: graalvm@20.3.0
```

Thanks for the update :+1: Until the PR on jabba is merged, we could try the latest available version and see if this breaks as it does for you locally.